modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/TcsController.scala

```diff
@@ -281,16 +281,14 @@ object TcsController {
   final case class GuidersEnabled(
     pwfs1: GuiderSensorOptionP1, 
     pwfs2: GuiderSensorOptionP2,
-    oiwfs: GuiderSensorOptionOI, 
-    aowfs: GuiderSensorOptionAO
```

Comment:
This was for the Altair detector. It must be used only at GN, so I removed it form the common configuration.